Porting/perlgov-team-update

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env perl
+use v5.30.0;
+use warnings;
+use JSON::PP;
+
+my $file = 'Porting/core-team.json';
+my $data = JSON::PP->new->decode(scalar `cat $file`);
```

Comment:
This will need to either be `decode_json` or set the `utf8` property in the decoder, since it's reading bytes.

Reply:
The specific failure mode here is: JSON contains a `\uXXXX` escaped character, as well as UTF-8 encoded characters. As a result `$data` will have a mix of UTF-8 encoded text and decoded characters and thus be impossible to correctly encode for the subsequent commands. Therefore when reading JSON bytes, the UTF-8 decoding step is required before (or as part of) decoding.

Reply:
alternatively, the file can be read with the :encoding(UTF-8) layer applied.

Reply:
I've done as Karen suggested.